synfig-studio/src/gui/app.h

```diff
@@ -313,6 +315,7 @@ class App : public Gtk::Main, private IconController
 
 	App(const synfig::String& basepath, int *argc, char ***argv);
 	virtual ~App();
+	void on_shutdown() ;
 
```

Comment:
@rodolforg I am using on_shutdown() like this ! And it is working perfectly according to build logs ! No error ! but it shouldn't work right ?? coz I am not overriding the function from the parent class ?! Can You please tell me why is it working ? and why is it showing error on using ``` override ``` keyword on this function ! 

Thanks :) Moving ahead !

Reply:
> And it is working perfectly according to build logs

It don't. If you add a breakpoint inside this method, you will see that it was never called. 

> and why is it showing error on using override keyword on this function !

Because there is no such method in GTK 3. 

When you reading the documentation, pay attention to the url:
https://developer.gnome.org/gtkmm/stable/classGtk_1_1Application.html

`stable` means GTK 4. Correct URL for GTK 3 should contain `3.24` in URL -> 
https://developer.gnome.org/gtkmm/3.24/classGtk_1_1Application.html